firebase-appdistribution/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,4 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <style name="Theme.Feedback" parent="Theme.MaterialComponents.Light.NoActionBar">
```

Comment:
Does "Theme.Feedback" get translated into "Theme_Feedback" like the value we set in the activity?

I think maybe we should just go with camelcase ("FeedbackTheme"). That lines up with the docs: https://developer.android.com/develop/ui/views/theming/themes#CustomizeTheme

Reply:
Done